model/project.rb

```diff
@@ -70,6 +72,9 @@ def soft_delete
     DB.transaction do
       access_tags_dataset.destroy
       access_policies_dataset.destroy
+      access_control_entries_dataset.destroy
+      DB[:applied_subject_tag].where(tag_id: subject_tags_dataset.select(:id)).delete
```

Comment:
Should we delete action and object tags too?

Reply:
Yes, that was an oversight, and looks like it wasn't in the main PR.  I updated this PR to include it, and will rebase the main PR on this PR.